apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/ExplanationOfBenefitE2E.java

```diff
@@ -706,10 +707,13 @@ public void searchEobByPatientIdWithLastUpdatedAndPagination() {
         .spec(requestAuth)
         .expect()
         .body("entry.size()", equalTo(3))
-        .body("total", equalTo(3))
+        .body("total", equalTo(expectedTotal))
         .statusCode(200)
         .when()
-        .get(nextLink);
+        .get(
+            URLDecoder.decode(
+                nextLink)); // Future: Decoder is a workaround, we should fix as part of a separate
+    // ticket.
```

Comment:
I created https://jira.cms.gov/browse/BFD-2883 for this; can you 
1. put this comment on a new line
2. link this bug here?
Thank you!

Reply:
Done